### ISSUE TITLE
[Feature Request] Add a Clock

### ISSUE THREAD

**Issue by kwadroke** [mozilla/hubs#644]
While using Hubs using a VR HMD it is hard to tell what time it is without taking off the HMD.

Add a simple clock so that people can see their local time while using Hubs while wearing a VR HMD.

**Comment by misslivirose** [comment]
A clock for HMD mode could definitely be useful. I would propose that it is only visible in HMD, not in desktop, but there should be a design proposal suggested before work on development starts.

**Comment by weddingdj** [comment]
A clock with a timer would be great. I could prototype something in Aframe... it sounds like a fun little project...

**Comment by emclaren** [comment]
We continue to get requests for this from the community.

**Comment by PauGuardiolaNet** [comment]
Yeah, this could be very very useful. Something like the watch that appears on the left hand of your virtual hand inside oculus:

![b0uk0ue2uffs](https://user-images.githubusercontent.com/66488666/101893426-0685b280-3ba5-11eb-8a0a-842794e53c4e.png)

**Comment by bsparks** [comment]
I would also want this feature. Additionally I think it'd be great if there could be a stop watch feature (maybe this is a separate request) so that as a event host you can control the time of the event. .. and even more feature creep from that, have a networked stop watch / timer in the room so everyone can see how long is left in the event. :)
